x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -427,14 +426,6 @@ describe('When on the Trusted Apps Page', () => {
       expect(queryByTestId('addTrustedAppFlyout-createForm')).not.toBeNull();
     });
 
-    it('should have list of policies populated', async () => {
```

Comment:
Why delete this test case? If I got this right, this test ins validating that the page passes along the set of policies received from the API call to fleet to the EffectPolicySelector component. You probably need to modify this existing test to (after render) click on the `perPolicy` button so that the list of policies can be shown.

Reply:
ah i guess that makes sense. i think i just deleted it because i figured the default would be the global policy which wouldn't show anything